crates/opencascade-sys/build.rs

```diff
@@ -39,6 +39,11 @@ fn main() {
         build.define("OCC_CONVERT_SIGNALS", "TRUE");
     }
 
+    if let "windows" = std::env::consts::OS {
```

Comment:
Sorry, last question:

Is there a reason we should use this instead of the `is_windows` variable? `is_windows` relates to the target, so I suppose we want to detect if windows is the _host_ OS, right?

Reply:
Yes

Reply:
Got it, thanks! And sorry for the huge delay on merging this one.